app/views/assessments/_partial_feedback_js.html.erb

```diff
@@ -0,0 +1,50 @@
+<%= external_javascript_include_tag "lodash" %>
+<script>
+  function refreshPartial() {
+    $.ajax({
+        url: `getPartialFeedback?job_id=${<%= @job_id %>}`,
+        success: (res) => {
+          if (!res) return;
+          if (res.partial_feedback) {
+            $('#partial-feedback').html(`
+              <pre>${res.partial_feedback}</pre>
+              <div class="autograding-in-progress">
+                <i class="refresh-feedback small material-icons" onclick="onClickRefresh()">refresh</i>
+                <pre>Autograding in progress</pre>
+              </div>`);
+            $('.feedback-status').html("In-Progress");
+            $('.feedback-status').removeClass("feedback-status__queued");
+            $('.feedback-status').addClass("feedback-status__inprogress");
+            $( "#result-body" ).load(window.location.href + " #result-body" );
+
+          } else if (res.queue_position != null) {
+            $('#partial-feedback').html(`
+              <div class="autograding-in-progress">
+                <pre>You are at position ${(res.queue_position) + 1} / ${(res.queue_length)} in the autograding queue.</pre>
+              </div>`);
+              $('#status-badge').html("Queued");
+              $('#status-badge').removeClass("feedback-status__inprogress");
+              $('#status-badge').addClass("feedback-status__queued");
+              $('#result-body').html(`
+              <h4>Submission queued...</h4>
+              <p>
+                Your submission is currently in the autograding queue, waiting to be graded.
+              </p>
+              <p>
+                Press the refresh button to get updates.
+              </p>`);
+          } else if (!res.is_assigned) {
+            /* if there's no partial feedback and no queue position,
+                refresh the page to see if full feedback is available */
+            location.reload();
```

Comment:
I think this is the line that is causing the refresh loop. Basically I think my tango is acting weird (partialfeedback endpoint was not setup properly, so it doesn't work), and thus there is no partial feedback and no queue position.
I don't think what we should do here is to reload again tho

Reply:
I have figured out how to replicate this bug:
- Set up Tango without the PR that introduces partial feedback
- Submit an assignment normally
- Because partial feedback isn't working (the endpoint doesn't exist), so it's confused causing this bootloop

Reply:
Basically, there should be error handling when the partial feedback endpoint throws an error. 
Here's what currently happens when autograding fails (other parts of it)
![image](https://user-images.githubusercontent.com/5773562/205335040-002ee284-55c5-47db-b75a-8684b0f113ac.png)